src/test/java/seedu/address/model/insurancepolicy/InsurancePolicyTest.java

```diff
@@ -0,0 +1,39 @@
+package seedu.address.model.insurancepolicy;
+
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertFalse;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+import org.junit.jupiter.api.Test;
+
+
+public class InsurancePolicyTest {
+    private static final String POLICY_ID_CONTAINS_CARROT = "Policy>1234>www.google.com";
```

Comment:
Would it be more appropriate to name this variable `POLICY_ID_CONTAINS_ANGULAR_BRACKETS` rather than `POLICY_ID_CONTAINS_CARROT`? Since carrot can refer to `^` too.

Reply:
You are right. Actually carrot probably refers to just ^. Will be changing to what you have suggested.